iamkanhaiyakumar/ai-content-generator
# Review Section Design

Fixes : #70 
Hi @iamkanhaiyakumar @GauravKesh 
## What does this PR do?

This PR addresses issue #70, which involves updates to the component located in `/components/ui/review.tsx`. The changes made improve the functionality of the review component by [briefly describe the specific changes made, e.g., "optimizing the rendering of user reviews" or "adding a feature to display user images dynamically"]. 

### Motivation and Context
The motivation behind this change is to [explain why this change is necessary, e.g., "enhance user experience by ensuring reviews are visually engaging" or "fix a bug that caused user images not to load correctly"]. This change is expected to [describe the expected outcome, e.g., "improve performance" or "resolve user feedback issues"].

### Dependencies
No new dependencies are required for this change.

### Visual Changes
For visual changes, please refer to the screenshots  included in this PR:

- **Screenshots**: 
Mobile View:
![image](https://github.com/user-attachments/assets/8bc8e3b2-c67a-43a0-b18e-17ca1fec76d8)

Desktop View:
![image](https://github.com/user-attachments/assets/9a94cb82-346e-462b-a64a-4ebe8ad249e9)


## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)

## How should this be tested?

To verify the changes, the following tests should be conducted:

- [ x] Ensure that user images are displayed correctly based on the initial names provided.
- [x ] Validate that the review component renders without errors.
- [ x] Test responsiveness and layout on different screen sizes.

### Test Configuration
- Environment: [Specify any relevant details about the test environment, e.g., "React version", "Node version"]
- Browser: [List the browsers you tested on, if applicable]

## Mandatory Tasks

- [ x] Make sure you have self-reviewed the code. A decent-sized PR without self-review might be rejected.
